qutebrowser/misc/cmdhistory.py

```diff
@@ -63,9 +63,9 @@ def __init__(self, history=None, parent=None):
         super().__init__(parent)
         self.handle_private_mode = False
         self._tmphist = None
-        if history is None:
-            self.history = []
-        else:
+
+        self.history = []
+        if history:
```

Comment:
What's the reason for this change? The idiom

```python
def foo(self, bar=None):
    if bar is None:
        self.bar = []
    else:
        self.bar = bar
```

Is commonly used in Python, and strictly speaking, your code doesn't do the same (though it doesn't matter in this case, I think) - if someone passes an empty custom container in it, `self.history` will be `[]` instead of that custom container.